### DISCUSSION TITLE
How to create element with linked relation

### DISCUSSION THREAD

**Discussion by rburgst** [keystonejs/keystone#7306] [Questions] [answered]
Lets assume the following schema
```graphql
type Work {
    id: ID
    title: String
    student: Student
  }

type Student {
  id: ID
  works: [Work!]  
}
```

When I now insert a new student and I want to already connect a work to it, I would need to do something like
```graphql
mutation {
  createStudent(data: {
    works: {
      create: [{title: 'foo', student: {connect: {id: ???}}]
    }
  })
```

**Comment by emmatown** [comment, marked as answer]
By creating `work` on a `Student` like that, each `work` item will in the `create` array will automatically be connected as this is relationship field.

You do not need explicitly provide a `connect` field.
